src/cluster/src/local.rs

```diff
@@ -32,6 +36,8 @@ pub struct LocalClusterInstallerBuilder {
     server_tls_policy: TlsPolicy,
     /// The TLS policy for the client
     client_tls_policy: TlsPolicy,
+    /// install system charts automatically
+    install_sys: bool,
```

Comment:
Maybe we should add a `local_chart: Option<PathBuf>` to the builder and later down, we have a `with_local_chart` function that assigns `self.local_chart = Some(local_chart)`.

Reply:
Yeah, I don't think specifying chart options is supported for local installations. There is option to specify chart location but that is under `k8_config` of InstallOpt. I am not sure if we want to extend that to local installations. 
@sehz Any comments?

Reply:
Chart is only used for kubernetes